packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md

```diff
@@ -38,9 +38,9 @@ For configuring a custom email address:
 
 You can customize your Okta organization by replacing the Okta domain name with your own domain name. Your customized domain allows you to create a seamless branded experience for your users so that all URLs look like your app.
 
-For example, you use Okta as a user store for your apps, but you don't want your users to know that the app uses Okta behind the scenes. Okta orgs host pages on subdomains such as `example.okta.com`. You can create a [CNAME record](https://datatracker.ietf.org/doc/html/rfc1035#section-3.2.2) for the Okta domain, allowing you to alias it to another subdomain that you own, like `login.example.com`.
+For example, you use Okta as a user store for your apps. But you don't want your users to know that the app uses Okta as its host. Okta orgs can host pages on subdomains such as `example.okta.com`. You can create a [CNAME record](https://datatracker.ietf.org/doc/html/rfc1035#section-3.2.2) for the Okta domain, allowing you to alias it to another subdomain that you own, like `login.example.com`.
```

Comment:
```suggestion
For example, you use Okta as a user store for your apps, but you don't want your users to know that the app uses Okta as its host. Okta orgs can host pages on subdomains such as `example.okta.com`. You can create a [CNAME record](https://datatracker.ietf.org/doc/html/rfc1035#section-3.2.2) for the Okta domain, allowing you to alias it to another subdomain that you own, like `login.example.com`.
```

Reply:
agree on removing "behind the scenes" but I'd keep the "but" in the same sentence for flow